.github/CODEOWNERS

```diff
@@ -675,7 +675,7 @@ modules/nf-core/ngscheckmate/fastq/** @sppearce
 modules/nf-core/ngscheckmate/ncm/** @sppearce
 modules/nf-core/ngscheckmate/patterngenerator/** @sppearce
 modules/nf-core/ngscheckmate/vafncm/** @sppearce
-modules/nf-core/nonpareil/** @jfy133
+modules/nf-core/nonpareil/nonpareil/** @jfy133
```

Comment:
I think you're also missing nonpareil curve

Reply:
When is that meant to come in? It wasn't added in my original PR for nonpareil...? But I can add it manually if it doesn't breka things